src/index.html

```diff
@@ -4,9 +4,48 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Roboto:ital,wght@0,500;0,700;1,500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar calendar--start-sun">
```

Comment:
```suggestion
      <div class="calendar calendar--start-day-sun">
```
The modifier shold be `start-day`, and also add the modifier `start-day` to change from what day start you calendar(mon, tue, wed, etc.)
![image](https://github.com/mate-academy/layout_calendar/assets/63243136/e03fd7ae-2f9c-426a-9c0b-cd2931e8ad92)